store/util_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"sync"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/kv"
+	"github.com/pingcap/tidb/store/mockstore/unistore"
+	"github.com/pingcap/tidb/store/tikv"
+)
+
+var (
+	withTiKVGlobalLock sync.RWMutex
+	WithTiKV           = flag.Bool("with-tikv", false, "run tests with TiKV cluster started. (not use the mock server)")
+	pdAddrs            = flag.String("pd-addrs", "127.0.0.1:2379", "pd addrs")
+)
+
+// NewTestStore creates a kv.Storage for testing purpose.
+func NewTestStore(c *C) kv.Storage {
```

Comment:
why not use `tikv.NewTestStore` directly?

Reply:
it's in _test.go file, we cannot import it.

Reply:
Could we move it to `tikv/test_util.go` so we can use it here?

Reply:
I'll give it a try.

Reply:
@AndreMouche Looks we cannot do it. The reason is that `NewTestStore` relies on the testing flags `with-tikv` and `pd-addrs`. If they are included in non-testing packages, the flags will appear in the flag list of `tidb-server`.

Reply:
sad